.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     name: test_ci
     strategy:
       matrix:
-        java-version: [ 11, 17 ]
+        java-version: [ 11, 17, 21 ]
```

Comment:
Is it worth to explode the test matrix by adding another Java version? Can't we test Java 21 compatibility without running all integration tests with it?

Reply:
it does not run all
it runs only against flink1.18 (same as for 17)
below there are exclusions for 1.16 and 1.17

I'm ok to reduce number of tests, however then would be great if you can provide a hint what else could be cut

Reply:
Oh! I missed that you were excluding. This is fine then.